src/UhdmAst.cpp

```diff
@@ -1695,6 +1695,39 @@ AstNode* process_typedef(vpiHandle obj_h, UhdmShared& shared) {
     return typedefp;
 }
 
+AstNode* handle_parameters(vpiHandle obj_h, UhdmShared& shared) {
```

Comment:
This name is easy to confuse with functions like `vpi_handle`. Maybe change to `get_parameters` following other more complex node handling functions?

Reply:
Ok, you're right.